static/js/mediastream/api.js

```diff
@@ -22,16 +22,22 @@
 "use strict";
 define(['jquery', 'underscore', 'ua-parser'], function($, _, uaparser) {
 
-	var Api = function(version, connector) {
+	var Api = function(version, connector, encryption) {
+		this.e = $({});
 		this.version = version;
 		this.id = null;
 		this.sid = null;
 		this.session = {};
 		this.connector = connector;
+		if (!encryption.initialize(this)) {
+			console.warn("Encryption services failed to initialize");
+			this.encryption = null;
+		} else {
+			console.log("Encryption services initialized");
+			this.encryption = encryption;
+		}
 		this.iids= 0;
 
-		this.e = $({});
-
```

Comment:
This moved up to the start of the constructor as the encryption code needs to register events.